assets/messages.yml

```diff
@@ -4,38 +4,104 @@ variables:
+          *快捷链接*：
+          📓 [漏洞报告指南（英文）|https://bugs.mojang.com/projects/MC/summary] -- 💬 [社区支持（英文）|https://discord.gg/58Sxm23] -- 📧 [客户服务（英文）|https://help.minecraft.net/hc/en-us/requests/new] -- ✍️ [反馈与建议（英文）|https://feedback.minecraft.net/] -- 📖 [游戏维基|https://minecraft-zh.gamepedia.com]
     - project: mcapi
       value: |-
           *Quick Links*:
-          📓 [Issue Guidelines|https://bugs.mojang.com/projects/MCAPI/summary] -- 💬 [Community Support|https://discord.gg/58Sxm23] -- 📧 [Customer Support|https://help.minecraft.net/hc/en-us/requests/new] -- ✍️ [Feedback and Suggestions|https://feedback.minecraft.net/] -- 📖 [Game Wiki|https://minecraft.gamepedia.com/Minecraft_Wiki]
+          📓 [Issue Guidelines|https://bugs.mojang.com/projects/MCAPI/summary] -- 💬 [Community Support|https://discord.gg/58Sxm23] -- 📧 [Customer Support|https://help.minecraft.net/hc/en-us/requests/new] -- ✍️ [Feedback and Suggestions|https://feedback.minecraft.net/] -- 📖 [Game Wiki|https://minecraft.gamepedia.com]
+      localizedValues:
+        de: |-
+          *Nützliche Links*:
+          📓 [Anleitung zum Melden von Fehlern|https://bugs.mojang.com/projects/MCAPI/summary] -- 💬 [Community Support|https://discord.gg/58Sxm23] -- 📧 [Kundensupport|https://help.minecraft.net/hc/de/requests/new] -- ✍️ [Feedback und Vorschläge|https://feedback.minecraft.net/] -- 📖 [Minecraft Wiki|https://minecraft-de.gamepedia.com]
+        es: |-
+          *Links rapidos*:
+          📓 [Como reportar problemas|https://bugs.mojang.com/projects/MCAPI/summary] -- 💬 [Soporte de la comunidad|https://discord.gg/58Sxm23] -- 📧 [Soporte oficial|https://help.minecraft.net/hc/es/requests/new] -- ✍️ [Sugerencias y opiniones|https://feedback.minecraft.net/] -- 📖 [Wiki|https://minecraft-es.gamepedia.com]
+        zh: |-
+          *快捷链接*：
+          📓 [漏洞报告指南（英文）|https://bugs.mojang.com/projects/MCAPI/summary] -- 💬 [社区支持（英文）|https://discord.gg/58Sxm23] -- 📧 [客户服务（英文）|https://help.minecraft.net/hc/en-us/requests/new] -- ✍️ [反馈与建议（英文）|https://feedback.minecraft.net/] -- 📖 [游戏维基|https://minecraft-zh.gamepedia.com]
     - project: mcd
       value: |-
           *Quick Links*:
-          📓 [Issue Guidelines|https://bugs.mojang.com/projects/MCD/summary] -- 💬 [Community Support|https://discord.gg/58Sxm23] -- 📧 [Customer Support|https://help.minecraft.net/hc/en-us/requests/new] -- ✍️ [Feedback and Suggestions|https://feedback.minecraft.net/] -- 📖 [Game Wiki|https://minecraft.gamepedia.com/Minecraft_Wiki] -- 📖 [FAQs|https://help.minecraft.net/hc/en-us/articles/360041345271]
+          📓 [Issue Guidelines|https://bugs.mojang.com/projects/MCD/summary] -- 💬 [Community Support|https://discord.gg/58Sxm23] -- 📧 [Customer Support|https://help.minecraft.net/hc/en-us/requests/new] -- ✍️ [Feedback and Suggestions|https://feedback.minecraft.net/] -- 📖 [Game Wiki|https://minecraft.gamepedia.com] -- 📖 [FAQs|https://help.minecraft.net/hc/en-us/articles/360041345271]
+      localizedValues:
+        zh: |-
```

Comment:
German uses FAQ commonly as well (though I have never seen it with the plural s, in both languages.
I think the others you already have.

Reply:
:thinking: On the official website, [MCE](https://help.minecraft.net/hc/en-us/articles/360033744412-Minecraft-Earth-FAQs) and [REALMS](https://help.minecraft.net/hc/en-us/articles/360034670172-Minecraft-Realms-Plus-and-Minecraft-Realms-Java-Edition-FAQs-) uses `FAQs`, while [MCD](https://help.minecraft.net/hc/en-us/articles/360041345271) uses `FAQ`. I can see both spelling on the [wikipedia page](https://en.wikipedia.org/wiki/FAQ) so I guess it's not wrong to write `FAQs` here